lib/src/base-styles/_nx-forms.scss

```diff
@@ -81,3 +81,18 @@
   margin: 0 0 $nx-spacing-xl 0;
   padding: 0;
 }
+
+// Inline form group
+.nx-form-group--inline {
```

Comment:
Per my other comment, I don't think putting in in the form group makes sense. But if it did, I'd ask whether the modifier is really necessary as this is the only case we have so far where multiple things are in the same form group.

Reply:
> Per my other comment, I don't think putting in in the form group makes sense. But if it did, I'd ask whether the modifier is really necessary as this is the only case we have so far where multiple things are in the same form group.

https://github.com/sonatype/sonatype-react-shared-components/pull/164/commits/43f39c241667c908c92b52989983f40248909845